ntoskrnl/ke/i386/kiinit.c

```diff
@@ -159,7 +159,25 @@ KiInitMachineDependent(VOID)
             CpuCount = KeNumberProcessors;
             KeIpiGenericCall(Ki386EnableXMMIExceptions, (ULONG_PTR)&CpuCount);
 
-            /* FIXME: Implement and enable XMM Page Zeroing for Mm */
+            /* Patch the KeZeroPages function to enable fast zeroing */
+            PUCHAR Instr = (PUCHAR)KeZeroPages;
+            switch(*Instr)
+            {
+                case 0xE9:
+                    /* 32-bit jmp */
+                    Instr[4] = 0x90; //NOP
+                    Instr[3] = 0x90; //NOP
+                    Instr[2] = 0x90; //NOP
```

Comment:
I think just one mention of "nop" is sufficient 😅 
```suggestion
                    Instr[4] = 0x90; // NOP
                    Instr[3] = 0x90;
                    Instr[2] = 0x90;
```